tests/clickhouse-test

```diff
@@ -935,7 +935,7 @@ class TestCase:
 
         if (
             self.testcase_args.test_runs > 1
-            and total_time > 60
+            and total_time > 120
```

Comment:
This check does not really make sense because of ThreadFuzzer and the parallel execution of tests in the flaky check. 
For example: https://s3.amazonaws.com/clickhouse-test-reports/45604/d3630c89e78fd3e421d143c141259e9c16799041/stateless_tests_flaky_check__asan_/run.log 
In most cases the `total_time` was about 10-12 sec, but in one case it was 157 sec. Increasing the threshold to 120 sec will not help. Maybe we should check average/median test execution time instead.